solidity/contracts/converter/BancorConverter.sol

```diff
@@ -824,6 +826,54 @@ contract BancorConverter is IBancorConverter, SmartTokenController, Managed, Con
         return bancorNetwork.convertForPrioritized3(_path, amount, _minReturn, msg.sender, _conversionId, _block, _v, _r, _s);
     }
 
+    /**
+        @dev ensures transfer of tokens, taking into account that some ERC-20 implementations don't return
+        true on success but revert instead
+
+        @param _token     the token to transfer
+        @param _to        the address to transfer the tokens to
+        @param _amount    the amount to transfer
+    */
+    function ensureTransfer(IERC20Token _token, address _to, uint256 _amount) private {
```

Comment:
how about moving these functions to a shared utility?

Reply:
definitely seems like the right idea, probably will move them to Utils